transform/transformers/pck_transformer.py

```diff
@@ -142,6 +142,21 @@ def round_currency_values(self):
                 for k, v in self.data.items()  # noqa
                     if k in self.rsi_currency_questions or k in self.qcas_currency_questions})  # noqa
 
+    def parse_negative_values(self):
+        """If any number field contains a negative value then replace it with a number containing
+        the maxiumum number of 9's that downstream will allow
+        """
+        for k, v in self.data.items():  # noqa
+            try:
+                # If v isn't a number then an exception is thrown and we skip it
+                int_v = int(v)
+                # If the original number is between -1 and -499, it gets rounded to -0.  In this case, we want it to
+                # also be all 9's as the original number was negative.
+                if v == '-0' or int_v < 0:
+                    self.data[k] = '9' * 11
+            except ValueError:
+                continue
```

Comment:
Just a suggestion. Based on the current set up, we define the currency questions at the top of the file, hence you could just pass in those and check that to see if they are in the `data.values()`, which avoid the needs to do a `try except`?